Mixpanel/Mixpanel.m

```diff
@@ -818,9 +827,11 @@ + (NSString *)libVersion
 - (NSDictionary *)collectAutomaticProperties
 {
     NSMutableDictionary *p = [NSMutableDictionary dictionary];
-    UIDevice *device = [UIDevice currentDevice];
     NSString *deviceModel = [self deviceModel];
+#if !defined(MIXPANEL_WATCH_EXTENSION)
+    UIDevice *device = [UIDevice currentDevice];
```

Comment:
i'll rewrite this section to be cleaner. the properties it uses UIDevice for are the same ones we get from MixpanelWatchProperties so didn't make sense to use WKInterfaceDevice here if we were adding the same properties later